### ISSUE TITLE
Combobox list nodes do not show expanded when Inspect focus on the dropdown list value

### ISSUE THREAD

**Issue by ghost** [dotnet/winforms#2877]
* .NET Core Version: .NET Core SDK 5.0.100-preview.1.20116.7 + the VSIX Inserted into Dev16 16.6.0 [29816.22.master]

* Have you experienced this same bug with .NET Framework?: Yes 

**Problem description:**

Combobox list nodes do not show expanded when Inspect focus on the dropdown list value

**More Info:**
1. This is not a regression issue, it's can reproduce in the .NET Framework project and Core 3.1 project.
2. This issue can reproduce when the property values is ComboBox control in the PropertyGrid.
3. This issue cannot reproduce by ComboBox control or the control type is ComboBox control in the DGV cell.

**Minimal repro:**

1. Run attached app.
2. Open the inspect tool.
3. Click the dropdown list after the property RightToLeftLayout, and select one node.
4. Observer the inspect.
[SeparateIssue.zip](https://github.com/dotnet/winforms/files/4222873/SeparateIssue.zip)

**Actual behavior:**
Inspect list node is not expanded.
![Actual](https://user-images.githubusercontent.com/53589182/74797011-0071be80-527f-11ea-8314-a588ce2338f7.png)

**Expected behavior:**
Inspect list node is expanded.
![Expected](https://user-images.githubusercontent.com/53589182/74796909-c0124080-527e-11ea-8d83-5547594c0558.png)

**Comment by merriemcgaw** [comment]
@M-Lipin does this show up in Accessibility Insights for Windows passes? If not, we may not have an issue here that must be solved in 5.0

**Comment by vladimir-krestov** [comment]
@merriemcgaw I couldn't find Accessibility Insights tests for this issue. This tool doesn't swear on this.
This issue affects on Narrator once (Quick focus reset on UseWaitCursor entry in the first time: gif). After it works correctly.
![mfLUVxcs20](https://user-images.githubusercontent.com/49272759/75248029-55b05180-57e4-11ea-8359-7585092ec6fa.gif)

I think we can fix it in the future

**Comment by merriemcgaw** [comment]
I think we'll wait until there is a product impact or we have some free cycles to dig too deeply into this one. I'll mark it for future, and accessibility. But if it's not blocking anyone I think we should be OK. It's an odd behavior but I think it shouldn't be a violation.

**Comment by Nora-Zhou01** [comment]
Verified with 17.3.0 Preview 3.0 [32619.15.main] from main branch, NET 5.0 and .NET 7.0  not repro this problem, Combobox list nodes can show expanded when Inspect focus on the dropdown list value, so closed this issue now.
![0620](https://user-images.githubusercontent.com/104609169/174573639-26e404db-a35a-483c-b809-c127bb28676c.png)
